integrations/cross_browsers/lambdatest_runner.py

```diff
@@ -74,7 +73,10 @@ def get_session_url_with_retries(self, session_id, retries=5, delay=2, timeout=3
                 session_data = response.json()
                 test_id = session_data['data']['test_id']
                 session_url = f"https://automation.lambdatest.com/test?testID={test_id}"
+                
                 return session_url
+
+            
```

Comment:
Can you make sure the empty lines are consistent? 
Check if we use `1` line or `2` lines after a `method` or `class`?